indico/modules/auth/templates/accounts.html

```diff
@@ -43,6 +44,7 @@
                     {% else %}
                         {{ form_header(form, orientation='vertical', classes='no-block-padding') }}
                         {{ form_rows(form) }}
+                        {{ template_hook('below-account-form', new_prefix=true, include_form_style=false) }}
```

Comment:
I'm not sure what `new_prefix` is supposed to mean, but it does not seem necessary since the code handling the template hook can simply check for `if user.local_identity`